app/views/recipes/_form.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= form_with(model: recipe) do |form| %>
+  <% if recipe.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(recipe.errors.count, "error") %> prohibited this recipe from being saved:</h2>
+
+      <ul>
+        <% recipe.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= form.label :name, class: "label" %>
+    <div class="control">
+      <%= form.text_field :name, class: "input" %>
+    </div>
+  </div>
+
+  <div class="field">
+    <%= form.label :preparation_time, class: "label" %>
+    <div class="control">
+      <%= form.text_field :preparation_time, type: "time", class: "input" %>
+    </div>
+  </div>
+
+  <div class="field">
+    <%= form.label :cooking_time, class: "label" %>
+    <div class="control">
+      <%= form.text_field :cooking_time, type: "time", class: "input" %>
+    </div>
+  </div>
+
+  <div class="field">
+    <%= form.label :description, class: "label" %>
+    <div class="control">
+      <%= form.text_area :description, class: "textarea" %>
+    </div>
+  </div>
+
+  <div class="field">
+    <%= form.label :public, class: "checkbox" %>
+    <%= form.check_box :public %>
+  </div>
+
```

Comment:
- [ ] When a user clicks on the `public checkbox`, he or she needs to see his or her items inside the public recipes so that every user should see all public recipes created. So please do not show `New Recipe`, `show edit and destroy` methods in the public recipes.

![rails14](https://user-images.githubusercontent.com/55635977/142576359-241f343f-5dec-4743-ab24-e00251ecea80.png)

Reply:
Fixed